tests/README.md

```diff
@@ -11,8 +11,7 @@ Running tests using docker
 Docker containers can be used for running kafka system tests locally.
 * Requirements
   - Docker 1.12.3 (or higher) is installed and running on the machine.
-  - Test require a single kafka_*SNAPSHOT.tgz to be present in core/build/distributions, as well as the system test libs.
-   This can be done by running ./gradlew clean systemTestLibs releaseTarGz
+  - Test require that Kafka, including system test libs, is built. This can be done by running ./gradlew clean systemTestLibs
```

Comment:
Interesting, so `releaseTarGz` was never needed? I had assumed that there was an actual reason why this was different when the system tests were run via docker.

Reply:
We've had the `systemTestLibs` target for quite a long time for this. There definitely shouldn't be a need for different targets between regular and Docker versions of the tests, so I think this just slipped in accidentally. If we do diverge, we're better off just getting them consistent than having different requirements.